README.md

```diff
@@ -1,70 +1,137 @@
+### Prerequisites <a name="prerequisites"></a>
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+  Node.js
+  React.js
+  ES6
+  Git
+  HTML/JavaScript linters
 
-### Code Splitting
+### Install <a name="install"></a>
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/code-splitting](https://facebook.github.io/create-react-app/docs/code-splitting)
+To install the project's dependencies, run:
 
-### Analyzing the Bundle Size
+```
+npm install
+```
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
+### Usage <a name="usage"></a>
 
-### Making a Progressive Web App
+To use, run:
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+```
+npm start
+```
 
-### Advanced Configuration
+### Run tests <a name="run-tests"></a>
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+To test this application, run:
 
-### Deployment
+```
+npm test
+```
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-### `npm run build` fails to minify
+## 👥 Authors <a name="authors"></a>
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+👤 **Uzair-Manzoor**
+
+- GitHub: [@githubhandle](https://github.com/Uzair-Manzoor)
+- Twitter: [@twitterhandle](https://twitter.com/UzairKiyani5555)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/uzair-manzoor-b69996115/)
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[A medium fedility wire-frame application]**
+- [ ] **[Math Magicians web application]**
+- [ ] **[Secure deployment Render]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contribution <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+Kindly give this project a star if you like it.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](/LICENSE) license.
+
```

Comment:
- [ ] It seems like the MIT link is broken as it's redirecting to a "404 not found" error page and this is because you don't have a license file in your repo, so please make sure to add one.